Dockerfile

```diff
@@ -16,26 +16,26 @@ RUN apt-get -yqq update && apt-get -yqq install \
       gcc \
       git-core \
       gosu \
-      libmysqlclient-dev          `# Needed for MySQL-python` \
+      default-libmysqlclient-dev  `# Needed for mysqlclient` \
+      build-essential             `# Needed for mysqlclient` \
       libpq-dev \
-      python2 \
-      python2.7-dev \
+      python3.10 \
```

Comment:
Have you thought about simplifying this to `python3`? It would minimize the changes if in the future we switch to Ubuntu 24.04, for example.